tox.ini

```diff
@@ -1,32 +1,27 @@
 [tox]
 minversion = 3.10
 envlist = py310,py39,py38,py37,lint
-skipsdist = True
```

Comment:
This was causing CI to fail. I don't see a particular reason we'd need to set it to True.